python/src/scipp/plot/controller3d.py

```diff
@@ -14,11 +15,23 @@ class PlotController3d(PlotController):
     It handles some additional events from the cut surface panel, compared to
     the base class controller.
     """
-    def __init__(self, *args, pixel_size=None, positions=None, **kwargs):
+    def __init__(self,
+                 *args,
+                 pixel_size=None,
+                 positions=None,
+                 aspect=None,
+                 **kwargs):
 
         super().__init__(*args, **kwargs)
         self.positions = positions
         self.pixel_size = pixel_size
+        self.aspect = aspect
+        if self.aspect is None:
+            self.aspect = config.plot.aspect
+        if self.aspect not in ["equal", "auto"]:
```

Comment:
Aspect ratio feature really nice. However, we cannot control the aspect ratio in a single dimension. Say we have some part of the instrument in z beam direction very much spaced out from a 2d detector x,y, but you want to keep aspect ratio for x,y. Had you considered that - would that be something to consider later?

Reply:
We don't have the options of stretching the figure in 2d plots either, just "equal" or "auto", so i made it the same here.
So you are saying you would want to have something like `aspect={'x': 1.0, 'y': 1.0, 'z': 3.0}` to stretch the box in the z direction?
I would defer this to later work.

Reply:
I would defer this to later, but as you mention for the general plotting case, this highlights an issue actually, that this is **very important** for the 2D case. When working with SQW data, you often have different length scales. the delta E range for example can be very large compared to say the Qx range which in turn might be different to the Qz range.

Reply:
note that you can change the size/aspect ratio of 1d and 2d figures in the notebook by dragging the bottom right corner of the figure